services/navigation/builtin/builtin.go

```diff
@@ -93,10 +94,12 @@ func (conf *Config) Validate(path string) ([]string, error) {
 	}
 	deps = append(deps, conf.BaseName)
 
-	if conf.MovementSensorName == "" {
-		return nil, utils.NewConfigValidationFieldRequiredError(path, "movement_sensor")
+	if conf.MapTypeName == "GPS" {
```

Comment:
[note] Do we want to confirm no movement_sensor is present if map_type is not GPS (i.e. is "")